.github/workflows/manual-release.yml

```
@@ -0,0 +1,35 @@
+name: Manual Release
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)